package.json

```
@@ -39,7 +39,7 @@
   "dependencies": {
     "commander": "^6.0.0",
     "kleur": "^4.1.1",
-    "playwright-chromium": "^1.6.2",
+    "playwright-chromium": "=1.6.2",
```

Just specifying the exact version would pin the version. I wasn't sure about the `=` syntax since it's not mentioned in the npm docs.

```suggestion
    "playwright-chromium": "1.6.2",
```

---

Its part of semver https://github.com/npm/node-semver